app/models/concerns/settings_concern.rb

```diff
@@ -23,7 +23,7 @@ def enabled_component_ids
 
   # Disable/Enable components
   #
-  # @params [Array<String>] the ids of all the enabled components
+  # @param ids [Array<String>] the ids of all the enabled components
```

Comment:
oops looks like they both work and yardoc checks that the name matches the one used in the function.